package.json

```diff
@@ -115,7 +115,7 @@
     "@babel/preset-flow": "^7.12.1",
     "@babel/preset-react": "^7.12.7",
     "@babel/register": "^7.12.1",
-    "@skyscanner/bpk-foundations-web": "^4.3.1",
+    "@skyscanner/bpk-foundations-web": "^4.4.0",
```

Comment:
We should probably bump this to the latest v5 version to be consistent across the components?